src/ts/components/menuTab/MenuTab.ts

```diff
@@ -0,0 +1,38 @@
+import { selectDom, selectDomAll, addEvent,  } from "../../utils/dom";
+
+type ConvertTemplate = (path: string) => void;
+
+class MenuTab {
+  convertTemplate: ConvertTemplate;
+  vendingmachineWrap: HTMLElement;
+
+  constructor({ convertTemplate }) {
+    this.convertTemplate = convertTemplate;
+    this.vendingmachineWrap = selectDom("#app");
+    addEvent(this.vendingmachineWrap, "click", this.handleMenuTab);
+  }
```

Comment:
```suggestion
  vendingmachineWrap: HTMLElement;

  constructor(private readonly convertTemplate: (path: string) => void) {
    this.vendingmachineWrap = selectDom("#app");
    addEvent(this.vendingmachineWrap, "click", this.handleMenuTab);
  }
```
생성자로 할당해줄 수 있어요! [플레이그라운드](https://www.typescriptlang.org/play?#code/MYGwhgzhAEBSCmATAltA3tAsAKGn6wA9gHYQAuATgK7BmEUAUO+L0ADhcgG5hnzQV4YRCRABPaMTABbeAC5o5TsQDm0ALzQA5AGUA1mIDuYEHvgUtzfAEp0AXxxW8dHZWSqG1uUvdq0T1kEyKgpiaDIAC2QIADopWQBuAIdsFJweCmgAKyRUTWJ4QzhczyTsHCJSQhB4GJBCFQYclBiXNw9rawToAHoe-SMTMwogA)를 참고해주세요~

Reply:
js로 컴파일시 private 는 없어지기 때문에 의미가 없지 않나요? readonly만 붙여주는건 어떻게 생각하시나요?
흠.. readonly도 마찬가지 인가요?

Reply:
private는 외부에서 접근할 수 없게 설정하는 접근지정자이고, readonly는 재할당을 못하게 하는데 사용해서 용도가 다릅니다! 

> js로 컴파일시 private 는 없어지기 때문에 의미가 없지 않나요?

혹시 playground를 보고 말씀해주신거라면 strict mode에서는 private를 사용할 수 없어 안보이는걸로 알고있어요 👀 
런타임 이전에 에러를 잡아내기 위해 타입스크립트를 사용하는거기때문에 컴파일 결과에 포인트를 두는것보다 왜 타입스크립트를 사용하는지 알아보시면 좋을거같아요 🙂